docs/src/main/asciidoc/storage.adoc

```diff
@@ -86,6 +86,13 @@ This type represents a GCS file, which has associated https://cloud.google.com/s
 The `createSignedUrl` method can also be used to obtain https://cloud.google.com/storage/docs/access-control/signed-urls[signed URLs] for GCS objects.
 However, creating signed URLs requires that the resource was created using service account credentials.
 
+[CAUTION]
+====
+As of v2.0.2+, the `GoogleStorageResource.getURL()` method returns the `Bucket` or `Blob` 's `selfLink` value, rather than attempting to convert the `URI` a `URL` object that nearly-always threw a `MalformedURLException`.
+This value is notably different from `GoogleStorageResource.getURI()`, which returns the more commonly used `gs://my-bucket/my-object` identifier.
+See https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/341[here] for more details.
```

Comment:
I don't know if we should link to the PR in the refdoc. It seems too low-level.
Maybe just mention that we're doing it to allow for `spring.resources.static-locations`.

Reply:
Done.